src/store/actions/authentication.ts

```diff
@@ -62,6 +63,25 @@ export const checkCredentials = ({ email, password }) => {
                 console.log('authentication error: ', error);
             });
 
+        let accesskey = await AsyncStorage.getItem('access_token');
+        console.log(accesskey);
+        let decoded = jwtDecode(accesskey);
+        console.log(decoded);
+        let id_string = decoded.sub.split('|')[1];
+        dummyId = parseInt(id_string);
```

Comment:
instead of using id_string --> dummyId, maybe to make it more clear we can name it:
`const dummyId = parseInt(decoded.sub.split('|')[1])`